chain-config/src/lib.rs

```diff
@@ -17,10 +21,28 @@ pub trait ChainConfigContract:
     }
 
     #[only_owner]
-    #[endpoint(updateConfig)]
-    fn update_config(&self, new_config: SovereignConfig<Self::Api>) {
+    #[endpoint(updateSovereignConfig)]
+    fn update_sovereign_config(
+        &self,
+        hash_of_hashes: ManagedBuffer,
+        new_config: SovereignConfig<Self::Api>,
```

Comment:
This is not ok.
The old endpoint was fine because can to be called from sovereign-forge.

You need to create a different endpoint:
- should not be `only_owner` because is called by the bridge service
- just use `self.require_setup_complete();`
- lock_operation_hash
- `require_valid_config` should not fail the transaction because we need to generate event
- set config if valid
- remove_executed_hash
- generate `executedBridgeOp` event

Reply:
Maybe refactor `require_valid_config` function to return `(bool, string)` [is_valid, err_message]
- for the old endpoint you can use `sc_panic` if not valid
- for this endpoint, we need to generate a different event with the error message